examples/pytorch/dimenet/README.md

```diff
@@ -0,0 +1,105 @@
+# DGL Implementation of DimeNet
+
+This DGL example implements the GNN model proposed in the paper [Directional Message Passing for Molecular Graphs](https://arxiv.org/abs/2003.03123) and [Fast and Uncertainty-Aware Directional Message Passing for Non-Equilibrium Molecules](https://arxiv.org/abs/2011.14115). For the original implementation, see [here](https://github.com/klicperajo/dimenet).
+
+Contributor: [xnuohz](https://github.com/xnuohz)
+
+### Requirements
+The codebase is implemented in Python 3.6. For version requirement of packages, see below.
+
+```
+click 7.1.2
+dgl 0.5.3
+logzero 1.6.3
+numpy 1.19.5
+pandas 1.1.5
+ruamel.yaml 0.16.12
+scikit-learn 0.24.1
+scipy 1.5.4
+sympy 1.7.1
+torch 1.7.0
+tqdm 4.56.0
+```
+
+### The graph datasets used in this example
+
+The DGL's built-in QM9 dataset. Dataset summary:
+
+* Number of Molecular Graphs: 130,831
+* Number of Tasks: 12
+
+### Usage
+
+**Note: DimeNet++ is recommended to use over DimeNet.**
+
+###### Model options
+```
+emb_size          int    Embedding size used throughout the model.                              Default is 128
+out_emb_size      int    Output embedding size used in DimeNet++.                               Default is 256
+int_emb_size      int    Input embedding size used in DimeNet++.                                Default is 64
+basis_emb_size    int    Basis embedding size used in DimeNet++.                                Default is 8
+num_blocks        int    Number of building blocks to be stacked.                               Default is 6   
+num_bilinear      int    Third dimension of the bilinear layer tensor in DimeNet.               Default is 8   
```

Comment:
"DimeNet" or "DimeNet++"?

Reply:
I put the configuration parameters of the two models together, so num_bilinear is a parameter of DimeNet.

Reply:
If you expect a user to use the scripts for both DimeNet and DimeNet++, you should separately list the commands for DimeNet and DimeNet++ and state which arguments are specifically for DimeNet or DimeNet++.